app/pages/(company)/about.vue

```
@@ -1,26 +1,27 @@
-        <div class="prose text-sm p8 sm:(text-base p2)">
-            <p>{{ t('about.paragraphs.introduction') }}</p>
-            <p>{{ t('about.paragraphs.mission') }}</p>
-            <p>{{ t('about.paragraphs.goal') }}</p>
-            <p>{{ t('about.paragraphs.importance') }}</p>
-            <p>{{ t('about.paragraphs.closing') }}</p>
-        </div>
-    </div>
-</template>
-
 <script setup lang="ts">
 const { t } = useI18n({
```

ESLint

[error] 'useI18n' is not defined.

(no-undef)